src/development/add-to-app/ios/project-setup.md

```diff
@@ -225,19 +248,15 @@ as a dependency manager in your existing applications.
 You must run `flutter build ios-framework`
 every time you make code changes in your Flutter module.
 
-If you're using the previous
-[Embed with CocoaPods and Flutter tools][] method,
-you can skip these instructions.
-
 The following example assumes that you want to generate the
-frameworks to `some/path/MyApp/Flutter/`.
+frameworks to `some/path/Flutter/`.
```

Comment:
I kind of liked the old `MyApp` directory so you know it should be relative or inside your host app in a known location, not just floating somewhere in another directory.

Reply:
The reason why I moved it out was because I thought it wouldn't be directly inside your app codebase because I figured if other developers didn't have dart / flutter on their computer then it wouldn't make sense for them to bring the code in when they're working on the app. But I can definitely add it back!

Reply:
This is the path to the built Flutter/App/plugin frameworks, not the Flutter module containing the dart code.  Without these frameworks the app won't build, so they need to make it available to other developers that don't have Flutter installed somehow.

Reply:
Ahh right, okay makes sense -- I'll change back!